app/state_machines/configurable_state_machine/config_validator.rb

```diff
@@ -1,6 +1,5 @@
 module ConfigurableStateMachine
 
-  #rubocop:disable Metrics/ClassLength
```

Comment:
Yeah I'd leave these in rather than changing the global rule 👍

Reply:
Done